Makefile.am

```diff
@@ -20,8 +20,13 @@ srpm:
 	$(MAKE) dist
 	rpmbuild $(RPM_DIRS) -bs $(PACKAGE_NAME).spec
 
+test:
+	$(MAKE) -C test/ test $($(ARGS))
+
```

Comment:
These lines are not necessary if port `test/Makefile.am` to Autotools Tesst[1]
```bash
check-local:
	PATH=${abs_top_srcdir}/src:$(PATH) RETRACE_SERVER_PLUGIN_DIR=${abs_top_srcdir}/src/plugins RETRACE_SERVER_CONFIG_PATH=${abs_top_srcdir}/src/config/retrace-server.conf $(PYTHON) run_test.py $(ARGS)
```

1: https://www.gnu.org/software/automake/manual/html_node/Tests.html#Tests